src/row.rs

```diff
@@ -186,6 +186,18 @@ pub struct Row<'stmt> {
     pub(crate) stmt: &'stmt Statement<'stmt>,
 }
 
+impl<'stmt> fmt::Debug for Row<'stmt> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let mut row = Vec::new();
+        let columns = self.stmt.columns();
+        for (count, c) in columns.iter().enumerate() {
+            // unwrap shouldn't panic since count will always be in range here?
+            row.push((count, c, self.get::<_, Value>(count).unwrap()))
```

Comment:
I think this could still probably panic if you had something like non-utf8 strings in the DB (which Value might have trouble with). Probably better to just return a fmt::Error in that case.

Reply:
The [`std::fmt`] document says

> Formatting implementations should ensure that they propagate errors from the Formatter (e.g., when calling write!). However, they should never return errors spuriously. That is, a formatting implementation must and may only return an error if the passed-in Formatter returns an error.

It may be surprise if panic when using `dbg!`.

How about using [`String::from_utf8_lossy`] or others for non-utf8 strings?

[`std::fmt`]: https://doc.rust-lang.org/std/fmt/index.html#formatting-traits
[`String::from_utf8_lossy`]: https://doc.rust-lang.org/std/string/struct.String.html#method.from_utf8_lossy

Reply:
Yes, I agree that we should not panic in a formatter.